automerge/tests/helpers/mod.rs

```diff
@@ -6,12 +6,12 @@ use std::{
 
 use serde::ser::{SerializeMap, SerializeSeq};
 
-pub fn new_doc() -> automerge::Automerge {
-    automerge::Automerge::new_with_actor_id(automerge::ActorId::random())
+pub fn new_doc() -> automerge::AutoTxn {
```

Comment:
I think it might be worth changing these tests to use the transaction API rather than the `AutoTxn` API. There are several tests which I'm not really sure are semantically equivalent with the `AtoTxn` API because the boundaries of the changes are different. This may be quite manual work though which I am happy to take on. I.e. this is more of a note to myself than a request for a change in this PR 🙂